index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>My portfolio</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href="./assets/libraries/fontawesome/css/all.css" rel="stylesheet">
+        <link rel="stylesheet" href="./assets/css/style.css">
+    </head>
+    <body>
+        <div class="container">
+            <header class="header">
+                <div class="logo">
+                    My Logo
+                </div>
+                <div class="menu">
+                    <i class="fas fa-bars"></i>
+                </div>
+            </header>
+            <div class="main-content">
+                    <h2 class="title">I'm Richard Glad to see you!</h2>
```

Comment:
- [ ] I think you could use the `<h1>` tag here, instead of the `<h2>` since this is the most important title on your page. Remember that when using `<hX>` tags, you should follow an order for good practices (and better SEO) :wink: